### PR TITLE
completed all tasks according to the conditions

### PR DESCRIPTION
**Pull request by gittafia**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -44,11 +44,57 @@ for (let i = 0; i < arr2.length; i++) {
...
     console.log(arr3[i]);
   }
 }
+
+//lesson-3
+//1.
+function palindrome(str) {
+  str = str.toLowerCase();
+  return str === str.split('').reverse().join('');
+}
```
Comment (Dronom) [inline main.js:61]: it will be great if your functions be clean. don't change your arguments, create new variable for that.

File: main.js [hunk 2]
```diff
@@ -44,11 +44,57 @@ for (let i = 0; i < arr2.length; i++) {
...
+
+//max
+function max(a, b) {
+  return a > b ? a : b;
+}
+console.log(max(200, 0));
+
+function max(a, b) {
+  return a > b ? a : b;
+}
```
Comment (Dronom) [inline main.js:84]: what is the difference?

File: main.js [hunk 3]
```diff
@@ -44,11 +44,57 @@ for (let i = 0; i < arr2.length; i++) {
...
+//3.
+function replaceZeros(array) {
+  for (let i = 0; i < array.length; i++) {
+    let numStr = '' + array[i];
+    if (numStr.indexOf('0') !== -1) {
+      numStr = numStr.replaceAll('0', 'zero');
+      array[i] = numStr;
+    }
+  }
+}
```
Comment (Dronom) [inline main.js:96]: function don't have return

File: main.js (hunk 1 above)
Comment (gittafia) [reply main.js:61]: TY. All fixes wil be in lesson-4.

File: main.js (hunk 2 above)
Comment (gittafia) [reply main.js:84]: TY. All fixes wil be in lesson-4.

File: main.js (hunk 3 above)
Comment (gittafia) [reply main.js:96]: TY. All fixes wil be in lesson-4.

**Pull request merged**